server/routes/auth.py

```diff
@@ -108,6 +111,28 @@ def post(self):
         return {"message": 'Successfuly update password'}, 200
 
 
+#PUT IN YOUR ENV FILE 
+#YELP_API_KEY = "I4VEQwNoLWKDbxaFK9Z-eN8W2L6Z9woSCz7Orp0DQCYzh8zePaKhMPnNQrOpM8Ij9myS3jCnG2n_WrnAgq2TZc5fw7KBtWrMvm5y7PigYrl_ArfZ7eLX01h6BXeSZHYx"
```

Comment:
You may want to remove the api key from here and add an instruction in the README for everyone to create their own api key and add it into the .emv